docs/usage/key.rst

```diff
@@ -35,9 +35,12 @@ Fully automated using environment variables:
 
 ::
 
-    $ BORG_NEW_PASSPHRASE=old borg init -e=repokey repo
+    $ export BORG_PASSPHRASE='old'
+    $ borg init -e=repokey repo
     # now "old" is the current passphrase.
-    $ BORG_PASSPHRASE=old BORG_NEW_PASSPHRASE=new borg key change-passphrase repo
+    $ export BORG_PASSPHRASE='old'
+    $ export BORG_NEW_PASSPHRASE='new' 
+    $ borg key change-passphrase repo
     # now "new" is the current passphrase.
```

Comment:
some thoughts:

did you try whether your new version works (esp. the borg init part)?

new line 38 and 41 are now duplicate, so 41 is not really needed and could be remove.

but, due to now the env vars being exported, it is not as clear as before any more, what the command is actually making use of.

so, if the bugfix part of this actually works and the original version does not, I'ld prefer a PR that **only** changes this and does not refactor to multiline and usage of export.

Reply:
I had the feeling that the previous version didn't work at all, aka this :
`BORG_NEW_PASSPHRASE=old borg init -e=repokey repo `
beeing executed without quote, would just ignore the "borg" command. So I directly try MY version and it works well.

Regarding the duplicate 38/41 : II would indeed, keep only the line 38. The duplicate make it more clear IMO

Reply:
```
$ FOO='bar' bash -c 'echo $FOO'
bar
user@remote:~$ FOO=bar bash -c 'echo $FOO'
bar
```

Maybe something special about your shell?

Reply:
I really do not want the exports there. Besides the scope issue in the example, there can also be scope issue when people use it.

After doing `$ export BORG_PASSPHRASE='old'` and `$ export BORG_NEW_PASSPHRASE='new'` these env vars will be active until the shell terminates. If the user later runs some other "borg key change-passphrase", e.g. on another repo, it will just use these env vars and the user might wonder why borg did not ask.

So, no `export`, but the bugfix part is welcome!